homeassistant/components/blink/__init__.py

```diff
@@ -60,7 +61,9 @@ def _blink_startup_wrapper(entry):
         no_prompt=True,
         device_id=DEVICE_ID,
     )
-    blink.refresh_rate = entry.data[CONF_SCAN_INTERVAL]
+    blink.refresh_rate = entry.options.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL)
+
+    _LOGGER.warning("Blink refresh rate is %s", blink.refresh_rate)
```

Comment:
This should probably be removed or changed to debug or info level so it doesn't show up in the logs by default.